### ISSUE TITLE
Save game time compatibility

### ISSUE THREAD

**Issue by jaynus** [acemod/ACE3#1098]
This is a future work item for saved game compatability concerning time:

esteldunedain [8:07 AM] 
it's basically a if _newACE_time < ACE_time then _newACE_time = ACE_time

**Comment by esteldunedain** [comment]
To support savegames we need to additionally ensure PFH with time intervals survive savegames on CBA.

**Comment by jaynus** [comment]
#1099 and #1098  for future work items based on this after this is merged in for 3.1

**Comment by WarezzK** [comment]
Anything progress this issue? :) Just a little info please. Thx!

**Comment by commy2** [comment]
I'm hijacking this.
I think I now understand enough about how save games (and mission restarts...) work to finish this.

Requires some changes in CBA. For anyone waiting on this: my personal goal is to finish this with the release of the Expansion / Tanoa (, because the new editor changes some details).

ref: https://github.com/acemod/ACE3/pull/3150

**Comment by esteldunedain** [comment]
:+1:

**Comment by JoeSchr** [comment]
Is it possible to make ACE3 save game compatible by removing some lesser needed pbo's? If yes, which want are most likely responsible for breaking save games? ty

**Comment by commy2** [comment]
No. It's a core issue.

**Comment by commy2** [comment]
Fixed with #3516
